amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-studio.adoc

```diff
@@ -72,7 +75,7 @@ image::amazon-sqs-studio-global-config-new.png[Configure Global Element]
 When you provide a queue URL in the *Default Global Queue URL* field of the global element, the connector automatically creates the queue and sets the URL of this queue as the provided value. All Amazon SQS message processors that reference the global element perform operations using this queue URL.
```

Comment:
Is the Default Global Queue URL also known as Standard Queue that is explained in the index.adoc page? Just asking because it is then explained the FIFO queue steps, but the Studio Configuration doc does not mentions Standard queues...I guess this is related.

Reply:
Will add to the JIRA